rest-fights/src/test/java/io/quarkus/sample/superheroes/fight/rest/FightResourceIT.java

```diff
@@ -130,18 +142,35 @@ public class FightResourceIT {
 	@InjectWireMock
 	WireMockServer wireMockServer;
 
-	@InjectKafkaConsumer
-	KafkaConsumer<String, io.quarkus.sample.superheroes.fight.schema.Fight> fightsConsumer;
+	@InjectKafkaCompanion
+  KafkaCompanion companion;
+
+  private static Vertx vertx;
 
 	@BeforeAll
 	public static void beforeAll() {
 		OBJECT_MAPPER.setSerializationInclusion(Include.NON_EMPTY);
-	}
+    // Set Apicurio Avro
+    vertx = Vertx.vertx();
+    RegistryClientFactory.setProvider(new VertxHttpClientProvider(vertx));
+  }
+
+  @AfterAll
+  static void afterAll() {
+    Optional.ofNullable(vertx)
+      .ifPresent(Vertx::close);
+  }
 
 	@BeforeEach
 	public void beforeEach() {
 		// Reset WireMock
 		this.wireMockServer.resetAll();
+    // Configure Avro Serde for Fight
+    companion.setCommonClientConfig(Map.of("apicurio.registry.avro-datum-provider",
```

Comment:
Should this use the constant `AvroKafkaSerdeConfig.AVRO_DATUM_PROVIDER` instead of a hard-coded `String`?